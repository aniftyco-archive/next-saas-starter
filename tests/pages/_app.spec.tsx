import React, { FunctionComponent } from 'react';
import { render } from '@testing-library/react';
import _App from '../../src/pages/_app';

const NoopComponent: FunctionComponent = () => null;

test('_app', () => {
  const { asFragment } = render(
    <_App Component={NoopComponent} pageProps={{}} />,
    {}
  );
  expect(asFragment()).toMatchSnapshot();
});
