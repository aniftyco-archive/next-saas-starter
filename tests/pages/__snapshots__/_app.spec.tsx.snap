// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`_app 1`] = `<DocumentFragment />`;
